spec/AwsS3AdapterSpec.php

```diff
@@ -521,6 +520,14 @@ public function getMatchers()
     /**
      * @param $key
      */
+    private function make_it_404_on_has_object($key)
+    {
+        $this->client->doesObjectExist($this->bucket, $key)->willReturn(false);
+    }
+
+    /**
+     * @param $key
```

Comment:
agree, but then would be good idea to remove all params like this or add string type